tools/e133/E133StatusHelper.cpp

```diff
@@ -96,5 +96,51 @@ string StatusCodeToString(E133StatusCode status_code) {
   }
   return "Unknown E1.33 Status Code";
 }
+
+
+bool IntToConnectStatusCode(uint16_t input,
+                            E133ConnectStatusCode *connect_status_code) {
+  switch (input) {
+    case ola::e133::CONNECT_OK:
```

Comment:
how about:
```
 case ola::e133::CONNECT_OK:
 case ola::e133::CONNECT_SCOPE_MISMATCH:
.......
      *connect_status_code = input;
      return true;
```
?

Reply:
That's a very elegant solution.

Unfortunately that code doesn't solely exist in E1.33 land. Do you fancy a clean-up of this (once merged) and the other similar code (e.g. there's some in https://github.com/OpenLightingProject/ola/blob/master/common/rdm/RDMHelper.cpp )?

We'd also need to add OLA_FALLTHROUGH to cover us from other compiler warnings, but looks neat.

Reply:
I can have a look, yes. Ping me once you merged it!

Reply:
It's in now @kripton but I assume you got an automatic notification anyway...